src/test/java/org/betonquest/betonquest/utils/math/TokenizerTest.java

```diff
@@ -59,45 +66,39 @@ private static void withVariables(final Executable executable, final ProtoVariab
         }
     }
 
-    /**
-     * Create the TokenizerTest.
-     */
-    public TokenizerTest() {
-    }
-
     @Test
-    /* default */ void testTokenizeNull() {
+        /* default */ void testTokenizeNull() {
```

Comment:
We need to do something about this auto-formatting issue, although it should not be done in this PR. Maybe revert the formatting of this class?